src/main/java/appeng/container/implementations/ContainerCellWorkbench.java

```diff
@@ -219,17 +221,18 @@ private FuzzyMode getWorkBenchFuzzyMode()
 
 	public void partition()
 	{
+
 		final IItemHandler inv = this.getUpgradeable().getInventoryByName( "config" );
 
-		final IMEInventory<IAEItemStack> cellInv = AEApi.instance().registries().cell().getCellInventory(
-				this.getUpgradeable().getInventoryByName( "cell" ).getStackInSlot( 0 ), null,
-				AEApi.instance().storage().getStorageChannel( IItemStorageChannel.class ) );
+		final ItemStack is = this.getUpgradeable().getInventoryByName( "cell" ).getStackInSlot( 0 );
+		final IStorageChannel channel = is.getItem() instanceof IStorageCell ? ((IStorageCell)is.getItem()).getChannel() : AEApi.instance().storage().getStorageChannel( IItemStorageChannel.class );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)